js1

```diff
@@ -1,2 +1,2 @@
 Q1 - hello how are you?
-ans - 
+ans - hello i am sunil kumar from sunil14701 and i am here just for testing
```

Comment:
good change but can you also add your homestate

Reply:
yay !sure

Reply:
yay !sure
but why

Reply:
just for your conformation.